File: Data Structures & Algorithms/buy-and-sell-crypto/submission-1.java
class Solution {
    public int maxProfit(int[] prices) {
        int l = 0, r = 1, res = 0;
        while (r<prices.length) {
            if (prices[r] < prices[l]) {
                l = r;
            } else {
                int profit = prices[r] - prices[l];
                res = Math.max(profit,res);
            }
            r++;
        }
        return res;
        
    }
}

File: Data Structures & Algorithms/counting-bits/submission-1.cs
public class Solution {
    public int[] CountBits(int n) {
        int[] res = new int[n+1];
        for (int x=1; x<=n;x++) {
            for (int i=0;i<32;i++) {
                if ((x&(1<<i)) != 0) {
                    res[x]++;                }
            }
        }
        return res;
    }
}
